server/routes/creatingLicences/handlers/hdc/confirmCreate.ts

```diff
@@ -25,6 +26,11 @@ export default class ConfirmCreateRoutes {
       this.probationService.getProbationer(nomisId),
     ])
 
+    if (config.hdcLicenceCreationBlockEnabled) {
+      logger.error('Access denied to HDC licence creation GET due HDC licence not to be created in CVL')
+      return res.redirect('/access-denied')
+    }
```

Comment:
probably not worth changing but this could have been before the api calls as they won't be used but it doesn't really matter

Reply:
Ah yeah, seen Seb mentioned this too above - I do agree and have moved